libctru/source/services/fs.c

```diff
@@ -9,20 +9,30 @@
 #include <3ds/env.h>
 #include "../internal.h"
 
+#define FS_MAX_ARCHIVE_HANDLES 32
```

Comment:
I still don't see how anybody would want to bypass the session handle override for more than 4 archives. Note than in the previous system only a hardcoded bypass (for sdmc) was supported.